guides/common/modules/proc_removing-a-virtual-machine-upon-host-deletion.adoc

```diff
@@ -0,0 +1,37 @@
+[id="removing-a-virtual-machine-upon-host-deletion_{context}"]
+= Removing a virtual machine upon host deletion
+
+By default, when you delete a host provisioned by {Project}, {Project} does not remove the actual VM on the external compute resource.
+You can configure {Project} to remove the VM when deleting the host entry on {Project}.
+
+[NOTE]
+====
+If you do not remove the associated VM and attempt to create a new VM with the same resource name later, it will fail because that VM name already exists in the external compute resource.
+You can still re-register the existing VM to {Project}.
+====
+
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-removing-a-virtual-machine-upon-host-deletion[].
+
+.Prerequisites
+* Your {Project} account has a role that grants the `view_settings` and `edit_settings` permissions.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *Settings* > *Provisioning*.
+. Change the value of the *Destroy associated VM on host delete* setting to *Yes*.
+
+[id="cli-removing-a-virtual-machine-upon-host-deletion"]
+.CLI procedure
+* Configure this behavior by using Hammer:
```

Comment:
I am unsure about this wording but I don't have a better alternative.

Reply:
You could use the sentence in line 5 ("You can configure {Project} to remove the VM when deleting the host entry on {Project}.") with a couple small changes:

* You can configure {Project} to remove the VM when deleting the host entry on {Project} by using the UI.
* You can configure {Project} to remove the VM when deleting the host entry on {Project} by using the Hammer CLI tool.